x-pack/plugins/infra/public/plugin.ts

```diff
@@ -34,15 +34,16 @@ export class Plugin implements InfraClientPluginClass {
       registerFeatures(pluginsSetup.home);
     }
 
-    pluginsSetup.observability.observabilityRuleTypeRegistry.register(
+    pluginsSetup.triggersActionsUi.ruleTypeRegistry.register(createInventoryMetricAlertType());
       createInventoryMetricAlertType()
     );
 
     pluginsSetup.observability.observabilityRuleTypeRegistry.register(
       createLogThresholdAlertType()
     );
+    pluginsSetup.triggersActionsUi.ruleTypeRegistry.register(createMetricThresholdAlertType());
```

Comment:
This duplicates line 46 and also looks like an incorrect merge conflict resolution.

Reply:
Fixed in [commit](https://github.com/elastic/kibana/pull/106519/commits/53b55be9f0c4d79d73087587a5925674542e2bbd)